### PR TITLE
Add redis support

### PR DESCRIPTION
**Pull request by vermario**
This setup mirrors what is available with the drupal chart, only using the latest version of the redis bitnami chart.

I have done these same changes for the `next-drupal-starterkit` using a local branch, and there I seem to be able to connect to redis from node, using multiple pods: https://github.com/wunderio/next-drupal-starterkit/pull/182 

> There were some issues in using this together with the mongodb chart, but now that https://github.com/wunderio/frontend-project-k8s/pull/116 has been merged, we should be able to continue the review.

### CODE REVIEW

**Comment by vermario** [comment]
@Jancis hi! I noticed that some other PRs have now been merged regarding mongodb, does it mean that this PR could be unblocked?

**Comment by Jancis** [comment]
@vermario yes, it's merged now, can assess redis issue again

**Comment by vermario** [comment]
I have merged `master` back into this, let's see what happens.

(I have to refresh my brain on what the issue was exactly ... :) )

File: silta/silta.yml [hunk 1]
```diff
@@ -65,5 +65,10 @@ silta-release:
 varnish:
   enabled: true
 
+redis:
+  enabled: false
```
Comment (vermario) [inline silta/silta.yml:69]: The build went through this time!

I wonder though, should this be set to `enabled: true`?

Comment (vermario) [reply silta/silta.yml:69]: @Jancis  ^

Comment (Jancis) [reply silta/silta.yml:69]: for testing yes, but i doubt it's required for all out feature branches.
probably?
@Rade333 might have better opinion

Comment (vermario) [reply silta/silta.yml:69]: At least for the drupal k8 project it is set to false: 

https://github.com/wunderio/drupal-project-k8s/blob/fbffed4227e966bd0828cddabf362b72c112e127/silta/silta.yml#L16C1-L19C20

Would make sense here as well then, probably

Comment (Rade333) [reply silta/silta.yml:69]: Let's keep it as false

File: charts/frontend/Chart.yaml [hunk 2]
```diff
@@ -32,3 +32,7 @@ dependencies:
 - name: silta-release
   version: ^1.x
   repository: file://../silta-release
+- name: redis
+  version: 18.19.x
```
Comment (Rade333) [inline charts/frontend/Chart.yaml:36]: The latest seems to be 19.1.5. Could you please test with that? :)

**Review by Rade333 — CHANGES_REQUESTED** [review]
Overall looking good! Added two comments.

File: charts/frontend/Chart.yaml [hunk 3]
```diff
@@ -32,3 +32,7 @@ dependencies:
...
+  repository: https://charts.bitnami.com/bitnami
```
[the remark this answers is not in the document]
Comment (vermario) [reply charts/frontend/Chart.yaml]: Updated in 16947a7a6b52b4d054cce26efa519eff49711250

File: charts/frontend/Chart.yaml (hunk 2 above)
Comment (vermario) [reply charts/frontend/Chart.yaml]: Updated in 16947a7a6b52b4d054cce26efa519eff49711250

**Comment by vermario** [comment]
@Jancis @Rade333  hi! I have updated to the latest version here and also in my test pr in `next-drupal-starterkit`. Looks like it's working :)

**Review by Rade333 — APPROVED** [review]
Thank you @vermario !

**Comment by vermario** [comment]
@Jancis @Rade333 hi! how do we go forward with this? Should I merge it? But then there's a manual step to add these changes to the actual charts, right?

**Comment by Rade333** [comment]
> @Jancis @Rade333 hi! how do we go forward with this? Should I merge it? But then there's a manual step to add these changes to the actual charts, right?

Hey @vermario. During the next silta release I will merge this and then also copy it to the charts repo. Because of a bunch of absences and holidays, it looks like it will be Tuesday 21.5

**Comment by vermario** [comment]
| Hey @vermario. During the next silta release I will merge this and then also copy it to the charts repo. Because of a bunch of absences and holidays, it looks like it will be Tuesday 21.5

ok great! :)

**Pull request merged**